packages/grid/_modules_/grid/hooks/features/keyboard/useGridKeyboardNavigation.ts

```diff
@@ -66,59 +92,60 @@ export const useGridKeyboardNavigation = (
   const navigateCells = React.useCallback(
     (params: GridCellParams, event: React.KeyboardEvent) => {
       event.preventDefault();
-
+      const { colIndex } = params;
```

Comment:
you can get the rowIndex here as well. Otherwise you have to update the params type

Reply:
I answered that before. 

The `rowIndex` that comes from the params changes depending on the sorting. Getting it from the params messes up the keyboard navigation if sorting is applied.

Reply:
Yes there was an issue with the rowIndex when a column was sorted. It is fixed now

Reply:
The last time I tried it didn't work. I'll try it again.